### ISSUE TITLE
[bug] - should be _

### ISSUE THREAD

**Issue by Freed-Wu** [wenLiangcan/pip2pkgbuild#20]
```sh
pip2pkgbuild pre-commit-mirror-maker
makepkg -fsi
```

will get
```
==> Making package: python-pre-commit-mirror-maker 1.9.0-1 (Sat 11 Mar 2023 12:37:43 AM CST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading pre-commit-mirror-maker-1.9.0.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 404
==> ERROR: Failure while downloading https://files.pythonhosted.org/packages/source/p/pre-commit-mirror-maker/pre-commit-mirror-maker-1.9.0.tar.gz
    Aborting...
```

That is correct filename is `https://files.pythonhosted.org/packages/source/p/pre-commit-mirror-maker/pre_commit_mirror_maker-1.9.0.tar.gz`.

See <https://aur.archlinux.org/packages/python-pre-commit-mirror-maker>.

**Comment by Freed-Wu** [comment]
Like <https://github.com/NixOS/nixpkgs/pull/223897>, it depends the true package name.

**Comment by wenLiangcan** [comment]
Fixed by 0.3.3.
